src/brogue/Combat.c

```diff
@@ -346,10 +344,10 @@ void moralAttack(creature *attacker, creature *defender) {
         }
 
         if ((defender->info.abilityFlags & MA_CLONE_SELF_ON_DEFEND) && alliedCloneCount(defender) < 100) {
-            if (distanceBetween(defender->loc.x, defender->loc.y, attacker->loc.x, attacker->loc.y) <= 1) {
-                splitMonster(defender, attacker->loc.x, attacker->loc.y);
+            if (distanceBetween(defender->loc, attacker->loc) <= 1) {
+                splitMonster(defender, attacker->loc);
             } else {
-                splitMonster(defender, 0, 0);
+                splitMonster(defender, INVALID_POS);
```

Comment:
Note that this looks different: we used to pass `{0, 0}` but now we're passing `{-1, -1}` - but this is okay, because the original check was that `.x > 0 && .y > 0` to identify "invalid" inputs and now it's `isPosInMap` which checks `.x >= 0 && .y >= 0`.

So there's no observable difference in behavior. The old way worked fine, this is just more generally consistent - the cell (0, 0) is inside the map but will always be a wall, whereas (-1, -1) is definitely outside of the map and therefore an invalid coordinate.